engine/src/test/java/io/zeebe/engine/processing/streamprocessor/SkipFailingEventsTest.java

```diff
@@ -339,7 +339,7 @@ public void processRecord(
           return TypedRecordProcessors.processors(
                   zeebeState.getKeyGenerator(), processingContext.getWriters())
               .onCommand(ValueType.JOB, JobIntent.COMPLETE, errorProneProcessor)
-              .onEvent(ValueType.JOB, JobIntent.ACTIVATED, dumpProcessor);
+              .onCommand(ValueType.JOB, JobIntent.THROW_ERROR, dumpProcessor);
```

Comment:
Actually I think the test tries to verify that even when the complete throws an error/exception this will not blacklist the instance, which means events of this process instance should still be processed. You change this now to a command which are always processed right?

Reply:
If the instance is on the blacklist then the command is not processed. 

Looking at the test, it doesn't have a clear semantic (i.e. the element is completed after the job was activated :sweat_smile:)
It is not possible to write events anymore because they don't trigger the processing - only commands. 

I'm open to suggestions :smile:

Reply:
Hm not sure whether this is correct for jobs, since we have no process instance key?

Reply:
Ok in the test we have, but not in real :thinking:

Reply:
Btw it is a bit problematic that we just ignore the commands, without rejection we saw that in the last game day that we didnt get any response in operate.

Reply:
Ok so this works with your changes, still it is a bit unexpected :sweat_smile:

Reply:
Agree. We should improve the whole test class (when reworking the blacklist concept).